src/Sylius/Bundle/CoreBundle/Doctrine/ORM/OrderRepository.php

```diff
@@ -21,31 +20,39 @@
 class OrderRepository extends CartRepository implements OrderRepositoryInterface
 {
     /**
-     * Create customer orders paginator.
-     *
-     * @param CustomerInterface $customer
-     * @param array             $sorting
-     *
-     * @return Pagerfanta
+     * {@inheritdoc}
      */
     public function createByCustomerPaginator(CustomerInterface $customer, array $sorting = [])
```

Comment:
It's not the best name, I can't come up with the better one, but `create` verb is misleading.